pkg/api/common_test.go

```diff
@@ -109,3 +111,122 @@ func doCORSWrapperRequest(t *testing.T, queryHandler http.Handler, url, origin s
 	queryHandler.ServeHTTP(w, req)
 	return w
 }
+
+func TestValidateConfig(t *testing.T) {
+	fileContents, err := ioutil.ReadFile("common_test.go")
+	if err != nil {
+		t.Fatal("error reading file contents common_test.go")
+	}
+	fileContentsString := strings.TrimSpace(string(fileContents))
```

Comment:
If you reuse the function you are testing in the test, you wouldn't be actually testing it. While this can seem redundant, its the best way I know of actually testing the functionality.